override/override.sh

```diff
@@ -0,0 +1,6 @@
+# Note that these files are based off the files in:
+# https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson
+
+cp build.sbt ../build.sbt
+cp pom.xml ../pom.xml
+cp JSON.java ../src/main/java/io/swagger/client/JSON.java
```

Comment:
yeah, i think it will be fixed when i merge https://github.com/Clever/clever-java/pull/2 into this branch which I will do before merging this in